File: Python/Start/判断根结点是否等于子结点之和（Easy）.py
class TreeNode:
    def __init__(self, val = 0, left = None, right = Node):
        self.val = val
        self.left = left
        self.right = right
class Solution:
    def check_tree(self, root: Optional[TreeNode]) -> bool:
        return True if root.val == root.left.val + root.right.val else False

File: C/Start/判断根结点是否等于子结点之和（Easy）.c
typedef struct
{
    int val;
    TreeNode *left;
    TreeNode *right;
}TreeNode;

typedef enum
{
    false,
    true
}bool;

bool check_tree(TreeNode *root) 
{
    return root->val == root->left->val + root->right->val ? true : false;
}